guide/chinese/python/bool-function/index.md

```diff
@@ -6,27 +6,27 @@ localeTitle: Python Bool函数
 
 ## 参数
 
-它需要一个参数， `x` 。使用标准[真值测试程序](https://docs.python.org/3/library/stdtypes.html#truth)转换`x` 。
+它需要一个参数`x` 。使用标准[真值测试程序](https://docs.python.org/3/library/stdtypes.html#truth)转换`x` 。
 
-## 回报价值
+## 返回值
 
 如果`x`为false或省略，则返回`False` ;否则返回`True` 。
 
 ## 代码示例
 ```
 print(bool(4 > 2)) # Returns True as 4 is greater than 2 
- print(bool(4 < 2)) # Returns False as 4 is not less than 2 
- print(bool(4 == 4)) # Returns True as 4 is equal to 4 
- print(bool(4 != 4)) # Returns False as 4 is equal to 4 so inequality doesn't holds 
- print(bool(4)) # Returns True as 4 is a non-zero value 
- print(bool(-4)) # Returns True as -4 is a non-zero value 
- print(bool(0)) # Returns False as it is a zero value 
- print(bool('dskl')) # Returns True as the string is a non-zero value 
- print(bool([1, 2, 3])) # Returns True as the list is a non-zero value 
- print(bool((2,3,4))) # Returns True as tuple is a non-zero value 
- print(bool([])) # Returns False as list is empty and equal to 0 according to truth value testing 
+print(bool(4 < 2)) # Returns False as 4 is not less than 2 
+print(bool(4 == 4)) # Returns True as 4 is equal to 4 
+print(bool(4 != 4)) # Returns False as 4 is equal to 4 so inequality doesn't holds 
+print(bool(4)) # Returns True as 4 is a non-zero value 
+print(bool(-4)) # Returns True as -4 is a non-zero value 
+print(bool(0)) # Returns False as it is a zero value 
+print(bool('dskl')) # Returns True as the string is a non-zero value 
+print(bool([1, 2, 3])) # Returns True as the list is a non-zero value 
+print(bool((2,3,4))) # Returns True as tuple is a non-zero value 
+print(bool([])) # Returns False as list is empty and equal to 0 according to truth value testing 
 ```
 
 ![:rocket:](//forum.freecodecamp.com/images/emoji/emoji_one/rocket.png?v=2 "：火箭：") [运行代码](https://repl.it/CVCS/2)
```

Comment:
The 'rocket' is a broken image, I am not sure how to fix this.

Understand that this is an inline image: 
https://daringfireball.net/projects/markdown/syntax.text
> Inline image syntax looks like this:
>     ![Alt text](/path/to/img.jpg)
>     ![Alt text](/path/to/img.jpg "Optional title")

Should I just use [semicolon rocket semicolon] instead?

Reply:
I deleted this broken image because the English version had already deleted that.